recipes/excel-mcp-server/recipe.yaml

```diff
@@ -42,10 +42,6 @@ tests:
         - excel_mcp
       pip_check: true
       python_version: ${{ python_min }}.*
-  - requirements:
-      run:
-        - python ${{ python_min }}.*
-        - pip
```

Comment:
Looks like we do need this. I wonder if it is b/c of the script below.

Reply:
@ocefpaf reverted!